core/src/main/scala/kafka/server/ControllerApis.scala

```diff
@@ -195,6 +198,61 @@ class ControllerApis(val requestChannel: RequestChannel,
       requestThrottleMs => createResponseCallback(requestThrottleMs))
   }
 
+  def handleDeleteTopics(request: RequestChannel.Request): Unit = {
+    if (!config.deleteTopicEnable) {
+      if (request.context.apiVersion() < 3) {
+        throw new InvalidRequestException("Topic deletion is disabled.")
+      } else {
+        throw new TopicDeletionDisabledException()
+      }
+    }
+    val deleteTopicsRequest = request.body[DeleteTopicsRequest]
+    val nameToId = new mutable.HashMap[String, Uuid]
+    deleteTopicsRequest.data().topicNames().iterator().asScala.foreach {
```

Comment:
You can do `forEach` on the Java collection directly, right? A few other cases like that below.